core/services/LogSvc.cpp

```
@@ -0,0 +1,80 @@
+
+    module_log_files[module] = std::shared_ptr<FILE>(file, [](FILE* f) { fclose(f); });
+
+    loguru::add_callback(
+        module.c_str(),
+        [](void* user_data, const loguru::Message& message) {
+            const std::string* target_module = static_cast<const std::string*>(user_data);
+            std::string prefix = "[" + *target_module + "]";
+
+            std::string message_text(message.message);
+
+            if (message_text.find(prefix) == 0) {
```

Cppcheck (2.10-2)

[performance] Inefficient usage of string

(stlIfStrFind)